instrumentation/jsf/jsf-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jsf/JsfTracer.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jsf;
+
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.servlet.ServletContextPath;
+import io.opentelemetry.instrumentation.api.tracer.BaseTracer;
+import javax.faces.FacesException;
+import javax.faces.component.ActionSource2;
+import javax.faces.component.UIViewRoot;
+import javax.faces.context.FacesContext;
+import javax.faces.el.EvaluationException;
+import javax.faces.el.MethodNotFoundException;
+import javax.faces.event.ActionEvent;
+
+public abstract class JsfTracer extends BaseTracer {
+
+  private static String getSpanName(String expressionString) {
```

Comment:
Could you add either a comment explaining the format of the expressionString, or add a unit-test with concrete examples. Or both.

Reply:
I removed this method and used expression as span name as it is